auth/transports/simplified_http_transport.go

```diff
@@ -0,0 +1,112 @@
+package transports
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"sync"
+
+	"github.com/bsv-blockchain/go-sdk/auth"
+)
+
+// SimplifiedHTTPTransport implements the Transport interface for HTTP communication
+type SimplifiedHTTPTransport struct {
+	url         string
+	client      *http.Client
+	onDataFuncs []func(*auth.AuthMessage) error
+	mu          sync.Mutex
+}
+
+// SimplifiedHTTPTransportOptions represents configuration options for the transport
+type SimplifiedHTTPTransportOptions struct {
+	URL         string
+	PollEnabled bool
+	PollTimeout int
+}
+
+// NewSimplifiedHTTPTransport creates a new HTTP transport instance
+func NewSimplifiedHTTPTransport(options *SimplifiedHTTPTransportOptions) (*SimplifiedHTTPTransport, error) {
+	if options.URL == "" {
+		return nil, errors.New("URL is required for HTTP transport")
+	}
+
+	return &SimplifiedHTTPTransport{
+		url:    options.URL,
+		client: &http.Client{},
+	}, nil
+}
+
+// Send sends an AuthMessage via HTTP
+func (t *SimplifiedHTTPTransport) Send(message *auth.AuthMessage) error {
+	jsonData, err := json.Marshal(message)
+	if err != nil {
+		return fmt.Errorf("failed to marshal auth message: %w", err)
+	}
+
+	resp, err := t.client.Post(t.url, "application/json", bytes.NewBuffer(jsonData))
+	if err != nil {
+		return fmt.Errorf("failed to send HTTP request: %w", err)
+	}
+	defer resp.Body.Close()
+
+	if resp.StatusCode < 200 || resp.StatusCode >= 300 {
+		body, _ := ioutil.ReadAll(resp.Body)
```

Comment:
ioutil is deprecated

```suggestion
		body, _ := io.ReadAll(resp.Body)
```